docs/src/upgrade/release-4.x.x.asciidoc

```diff
@@ -292,6 +293,39 @@ PDT. Primitive PDTs are string-based representations of a primitive type support
 contain a map of fields. You should consult your provider's documentation to determine what types of fields a
 particular PDT may contain.
 
+==== Changes to Authentication and Authorization
+
+With the move to HTTP, the only authentication option supported out-of-the-box is HTTP basic access authentication
+(username/password). The SASL-based authentication mechanisms are no longer supported (e.g. Kerberos). Your graph
+system provider may choose to implement other authentication mechanisms over HTTP which you would have to use via a
+request interceptor. Refer to your provider's documentation to determine if other authentication mechanisms are
+available.
+
+==== Transactions Disabled
```

Comment:
hmmm, this is true of M1, but not what will be 4.0 GA. Are these upgrade docs for the M1 or for 4.0 GA? i kinda feel like its the latter and that we need a different place to call out milestone limitations. how about adding:

> == TinkerPop 4.0.0.M1
> The 4.0.0.M1 is a milestone release. It is for meant as a preview version to try out the new HTTP API features in the the server and drivers. As this is a milestone version only, you can expect breaking changes to occur in future milestones for 4.0.0 on the way to its General Availability release.  The following sections detail important limitations and constraints pertinent to this milestone that may or may not apply to General Availability. 

as a new section and anything that is specific to M1 goes in there with

Reply:
I changed the version at the top from 4.0.0 to 4.0.0.M1 and added a disclaimer in that section. The disclaimer says that there was an attempt to tag parts that might change with an IMPORTANT box.